### PR TITLE
Education-Translations (Turkish) - Tested Exchanges

### PR DESCRIPTION
**Pull request by tigers40**
Projects\Foundations\Schemas\Docs-Topics\T\Tested\…\tested-exchanges-002-binance-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Teste…\tested-exchanges-003-ascendex-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\Test…\tested-exchanges-004-ftx-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Test…\tested-exchanges-005-binance-us-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\…\tested-exchanges-006-bitfinex-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\T…\tested-exchanges-007-kucoin-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\Tes…\tested-exchanges-008-luno-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\T…\tested-exchanges-009-kraken-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\T…\tested-exchanges-010-bittrex-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested…\tested-exchanges-011-bitstamp-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Te…\tested-exchanges-012-coinbasepro-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\Tes…\tested-exchanges-013-okex-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\…\tested-exchanges-014-gate.io-test-status.json
Projects\Foundations\Schemas\Docs-Topics\T\Tested\…\tested-exchanges-015-latoken-test-status.json
Projects\Foundations\Schema…\tested-exchanges-016-binance-futures-usds-m-coin-m-test-status.json

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        "updated": 1611662214310,
+        "translations": [
+            {
+                "language": "RU",
+                "text": "Индикатор выдает стандартную 14 настройку индекса относительной силы.",
+                "updated": 1643529696008
+            },
+            {
+                "language": "TR",
+                "text": "Özet: Gösterge, Göreceli Güç Endeksi'nin standart 14 ayarını üretir.",
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: Didn`t this page exist before or did you just add it back after it got deleted? :+1:

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: All files look great, when you answered and everything seems fine I will merge. :+1:

Comment (tigers40) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: Two of the same files appear in my repository. But I don't know why. Is this an issue that needs to be fixed?

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: Are they in the same folder? If yes and they have the exact same filename and content you could delete one and push again, that could fix the problem. If the names or paths are somehow different please let me know as we had that problem before a couple times. :+1:

Comment (tigers40) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: Yes, they are in the same folder and with the same names. I deleted the untranslated file.

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: The file is still showing up. Or was the other file not correct?

Comment (tigers40) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: The other file is not correct.

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: But now everything is correct from your side? :+1:

Comment (tigers40) [reply Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json:16]: yes, you can combine

**Review by gitforkgit — APPROVED** [review]

**Pull request merged**